kubectl-kuttle

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# start the sshuttle tunnel
+if [[ $1 == "start" ]]; then
+mkdir -p ~/.cache
```

Comment:
thanks for your contribution. can you add tabs or spaces to make the code more human readable?